support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.tsx

```diff
@@ -33,57 +38,24 @@ import {
 	setStripeSetupIntentClientSecret,
 	updateRecaptchaToken,
 } from 'pages/contributions-landing/contributionsLandingActions';
-import 'helpers/contributions';
-import type { ErrorReason } from 'helpers/forms/errorReasons';
-import { logException } from 'helpers/utilities/logger';
-import { trackComponentLoad } from 'helpers/tracking/behaviour';
-import type { IsoCountry } from 'helpers/internationalisation/country';
+import type {
+	State,
+	Stripe3DSResult,
+} from 'pages/contributions-landing/contributionsLandingReducer';
 import CreditCardsROW from './creditCardsROW.svg';
 import CreditCardsUS from './creditCardsUS.svg';
-import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import { routes } from 'helpers/urls/routes';
-import { InlineError } from '@guardian/src-user-feedback';
 import { StripeCardFormField } from './StripeCardFormField';
 import './stripeCardForm.scss';
-import { isValidZipCode } from 'helpers/forms/formValidation';
-// ----- Types -----//
 
-/* eslint-disable react/no-unused-prop-types */
-type PropTypes = {
```

Comment:
Where has this type definition gone?

Reply:
It's on line 101, using the `ConnectedProps` utility from Redux so the types can be inferred from what gets returned from `mapStateToProps` and `mapDispatchToProps`.